### ISSUE TITLE
Future of development

### ISSUE THREAD

**Issue by jeremyfelt** [Automattic/Custom-Javascript-Editor#24]
Howdy.

Is there a general roadmap for Custom Javascript Editor and/or is anybody still actively maintaining the project? I'm hoping to start using it in our environment and am trying to decide if forking or attempting to contribute changes directly upstream is the better option.

Any warnings on use in production are welcome as well. :)

Thanks!

**Comment by danielbachhuber** [comment]
I don't believe anyone is maintaining it, so YMMV. One thing to be aware of is there's no security model for JavaScript. If your users don't have `unfiltered_html`, they shouldn't have access to this plugin.

**Comment by jeremyfelt** [comment]
<blockquote>If your users don't have unfiltered_html, they shouldn't have access to this plugin.</blockquote>

I'm going with a custom capability and enforcing preliminary in person conversations about security. :)

**Comment by danielbachhuber** [comment]
You'll be fine then
